tests/utils/actions/misc.ts

```diff
@@ -119,20 +119,18 @@ async function initEmptyEditor({
         workspace.meta.pageMetas.forEach(meta => {
           const pageId = meta.id;
           const page = workspace.getPage(pageId);
-          if (page) {
-            initPage(page);
-          }
+          if (page) initPage(page).catch(console.error);
         });
-        workspace.slots.pageAdded.on(pageId => {
+        workspace.slots.pageAdded.on(async pageId => {
           const page = workspace.getPage(pageId);
           if (!page) {
             throw new Error(`Failed to get page ${pageId}`);
           }
-          initPage(page);
+          await initPage(page);
         });
       } else {
         const page = workspace.createPage({ id: 'page:home' });
-        initPage(page);
+        await initPage(page);
```

Comment:
what is the rule of when to use async/await and attaching .catch(console.error)?

Reply:
I have removed the console.error I found, it should not be the default behavior.

Reply:
The eslint rule is that we need to avoid floating promises...
1. either catch them or use async/await or void the promise
2. if a callback function is returning a promise, the calling function should be expecting a promise returning function

In this pr, I have assumed that if an async operation is taking place without await, it was intended to be done asynchronously in background so I have just caught the error with `console.error`